www/front_src/src/Resources/Actions/Resource/Downtime/Dialog.tsx

```diff
@@ -111,11 +118,11 @@ const DialogDowntime = ({
 
   class Adapter extends DayjsAdapter {
     public format(date, formatString): string {
-      return dayjs(date).tz(timezone).format(formatString);
+      return format({ date, formatString });
     }
 
     public date(value): dayjs.Dayjs {
```

Comment:
Only `Dayjs` could be imported, instead of default import

Reply:
We still use the default import right below (see the next method), so we need to import it this way anyway.